tmva/tmva/CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ if(CUDA_FOUND)
   add_dependencies(dnn_cuda move_headers)
 else()
   set(DNN_CUDA_LIBRARIES)
+  set(installoptions ${installoptions} OPTIONS REGEX "Cuda" EXCLUDE)
```

Comment:
Did you test whether that excludes both Architectures/Cuda.h and Architectures/Cuda?